src/components/Form.vue

```diff
@@ -19,20 +96,65 @@ export default {
       }
     }
   },
+
   methods: {
-    addTodo: function(evt) {
-      if (evt.keyCode === 13) {
+    submit: function() {
+      if (this.$store.state.todo.isEdit) {
+        if (this.$store.state.todo.addTodo.id == this.id) {
+          this.$store.commit("taskupdate", {
```

Comment:
i think if you simply write 
this.$store.commit("taskupdate", this.$store.state.todo.addTodo)  it will work. no need for these 5-6 lines